### PR TITLE
updated assets, regen refs

### PR DESCRIPTION
**Pull request by kbhawkey**
- incorporate changes from reference-docs (clean up of api generation, stylesheet changes so that left nav can scroll). Review the sizing of the main container.
- rebuilt API reference (new location of js files) from simplified directory structure and Makefile.
- rebuilt kubectl command reference. Apparently, there are *new* link errors. Changes were *not* made in the kubectl generator. Cleaned up the square brackets so that links resolve.

See issue #18444 
```
GENERATING AS:
[<a href="http://golang.org/pkg/text/template/#pkg-overview">]http://golang.org/pkg/text/template/#pkg-overview]</a>. </td>

FIXED AS:
[<a href="http://golang.org/pkg/text/template/#pkg-overview">http://golang.org/pkg/text/template/#pkg-overview</a>]. </td>
```

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page previews:

https://deploy-preview-18466--kubernetes-io-master-staging.netlify.com/docs/reference/generated/kubernetes-api/v1.17/

https://deploy-preview-18466--kubernetes-io-master-staging.netlify.com/docs/reference/generated/kubectl/kubectl-commands

**Comment by tengqm** [comment]
lgtm

File: update-imported-docs/Makefile_temp [hunk 1]
```diff
@@ -55,46 +54,35 @@ comp: cleancomp
...
 
-copycomp:
-	cp $(shell pwd)/gen-compdocs/build/* $(WEBROOT)/docs/reference/generated/
-
 # Build api docs
+
+create_version_dir:
+mkdir -p $(APISRC)/config/$(MINOR_VERSION)
+cp $(K8SROOT)/api/openapi-spec/swagger.json gen-apidocs/config/swagger.json
+cp gen-apidocs/config/$(MINOR_VERSION) - 1
```
Comment (kbhawkey) [inline update-imported-docs/Makefile_temp]: I was testing here. Will remove.

**Comment by kbhawkey** [comment]
A couple more link fixes were committed in reference-docs. Will rebuild the reference.

**Comment by kbhawkey** [comment]
Just regenerated API reference based on fixes:
https://github.com/kubernetes-sigs/reference-docs/pull/110
https://github.com/kubernetes-sigs/reference-docs/pull/114
https://github.com/kubernetes-sigs/reference-docs/pull/115
https://github.com/kubernetes-sigs/reference-docs/pull/116

Modified update-imported-docs.py to use K8S_RELEASE as a value such as `1.17` instead of the minor version (17). Testing versioning in Makefile.

**Comment by zacharysarah** [comment]
@kbhawkey 👋 Thanks for taking on these fixes, and for making them upstream in k-sigs/reference-docs!

Is this PR ready for review?

**Comment by kbhawkey** [comment]
> @kbhawkey wave Thanks for taking on these fixes, and for making them upstream in k-sigs/reference-docs!
> 
> Is this PR ready for review?

:wave: @zacharysarah . Yes, ready for review. I did use the updated Makefile.temp and update-imported-docs.py script to build the latest version of the reference, but I am still working on cleaning up the k-sigs/reference-docs/Makefile and using the Makefile.temp file to test the changes.

**Comment by zacharysarah** [comment]
Thanks for the review @tengqm. 

LGTM to me, too, even though it highlights how disparate our API experiences are. 

/lgtm
/approve

**Pull request merged**